C#/C#.sublime-syntax

```diff
@@ -79,6 +79,9 @@ contexts:
         - match: '\*/'
           scope: punctuation.definition.comment.end.cs
           pop: true
+        - match: ^\s*(\*)?(?!/)
```

Comment:
as the capture group is the only thing that receives a scope, why make it optional? it may make it harder for future maintainers to understand the purpose of consuming the leading whitespace...